src/main/kotlin/com/code_intelligence/cifuzz/config/TaskConfigurations.kt

```diff
@@ -75,6 +73,12 @@ private fun Project.registerPrintPackages(testSetAccess: TestSetAccess) {
     }
 }
 
+private fun Project.registerPrintTestSourceSets(testSetAccess: TestSetAccess) {
+    tasks.register("cifuzzPrintTestSourceSets", TestSourceSetsPrinter::class.java) { printTestSourceSets ->
```

Comment:
```suggestion
    tasks.register("cifuzzPrintTestSourceFolders", TestSourceSetsPrinter::class.java) { printTestSourceSets ->
```

...or `cifuzzPrintTestSources`

See my other comment about "source set" vs "source folders".